### PR TITLE
Changed the bidirectional traffic-flow check to unidirection in ixia pfcwd script for cisco-8000

### PR DESCRIPTION
**Pull request by rraghav-cisco**
Resubmitting instead of #6511.

Cisco-8000 platform forwards the rx traffic in a port that is also receiving Xoff. This was recently done. The ixia/pfcwd scripts assume that the bi-directional traffic will drop. This will fail. This PR fixes this.

What is the motivation for this PR?

When a port is receiving xoff, these a2a and m2o test cases assume that the receiving traffic will be dropped. But this asusmption is not valid for Cisco-8000. Due to this, the tests are failing. The check needs to be adjusted for cisco 8000 platform.
Also since the traffic is directly connected routes, we need to lower the CGM alpha value, so that the pfc response kicks in due to lower amount of memory available for queue.
How did you do it?
Added a check for cisco-8000 platform to verify only in the forwarding direction, not the reverse direction.
Also added a code to reduce the CGM value for cisco-8000 platform and revert it at the end of the script.

How did you verify/test it?
Ran it on my testbed, and verified that the failing cases are passing now.

@kevinskwang , pls check. I have kept only the required changes, and removed the other pre-submit related changes.

### CODE REVIEW

File: tests/ixia/pfcwd/test_pfcwd_m2o.py [hunk 1]
```diff
@@ -7,19 +7,22 @@
     ixia_api_serv_user, ixia_api_serv_passwd, ixia_api, ixia_testbed_config
 from tests.common.ixia.qos_fixtures import prio_dscp_map, all_prio_list,\
     lossless_prio_list, lossy_prio_list
+from tests.common.cisco_data import is_cisco_device
```
Comment (kevinskwang) [inline tests/ixia/pfcwd/test_pfcwd_m2o.py:10]: Don't see any reference of this funcion in the file.

File: tests/ixia/pfcwd/conftest.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import pytest
+from tests.common.cisco_data import is_cisco_device
+@pytest.fixture
+def setup_cgm_alpha_cisco(duthost):
+    if not is_cisco_device(duthost):
+        return
+    duthost.shell("mmuconfig -p pg_lossless_100000_300m_profile -a -6")
+    yield
+    duthost.shell("mmuconfig -p pg_lossless_100000_300m_profile -a -2")
```
Comment (kevinskwang) [inline tests/ixia/pfcwd/conftest.py:9]: The original value is "-2" on all cisco platform?

Comment (rraghav-cisco) [reply tests/ixia/pfcwd/conftest.py:9]: Yes, @kevinskwang . The default is -2.

File: tests/ixia/pfcwd/test_pfcwd_m2o.py (hunk 1 above)
Comment (rraghav-cisco) [reply tests/ixia/pfcwd/test_pfcwd_m2o.py:10]: Will remove it if not needed.

Comment (rraghav-cisco) [reply tests/ixia/pfcwd/test_pfcwd_m2o.py:10]: @kevinskwang , I have removed the import.

**Comment by rraghav-cisco** [comment]
@kevinskwang , I have updated the PR as per your comment.

File: tests/ixia/pfcwd/test_pfcwd_m2o.py (hunk 1 above)
Comment (kevinskwang) [reply tests/ixia/pfcwd/test_pfcwd_m2o.py:10]: I still can see this code line in the PR

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Failed<br/>- hook id: end-of-file-fixer<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/ixia/pfcwd/conftest.py<br/><br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/ixia/pfcwd/conftest.py:3:1: E302 expected 2 blank lines, found 0<br/>tests/ixia/pfcwd/files/pfcwd_multi_node_helper.py:5:1: F401 'tests.common.fixtures.conn_graph_facts.conn_graph_facts' imported but unused<br/>tests/ixia/pfcwd/files/pfcwd_multi_node_helper.py:5:1: F401 'tests.common.fixtures.conn_graph_facts.fanout_graph_facts' imported but unused<br/>tests/ixia/pfcwd/files/pfcwd_multi_node_helper.py:7:1: F401 'tests.common.ixia.ixia_fixtures.ixia_api_serv_ip' imported but unused<br/>tests/ixia/pfcwd/files/pfcwd_multi_node_helper.py:7:1: F401 'tests.common.ixia.ixia_fixtures.ixia_api_serv_port' imported but unused<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Review by kevinskwang — APPROVED** [review]

**Pull request merged**